C/CFITSIO/build_tarballs.jl

```diff
@@ -47,19 +55,26 @@ platforms = supported_platforms()
 
 # The products that we will ensure are always built
 products = [
-    LibraryProduct("libcfitsio", :libcfitsio)
+    LibraryProduct("libcfitsio", :libcfitsio),
+    ExecutableProduct("fpack", :fpack),
+    ExecutableProduct("funpack", :funpack),
+    ExecutableProduct("fitscopy", :fitscopy),
+    ExecutableProduct("fitsverify", :fitsverify),
+    ExecutableProduct("imcopy", :imcopy),
+    # Not available on Windows
+    # ExecutableProduct("smem", :smem),
+    # ExecutableProduct("speed", :speed),
 ]
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
+    Dependency("Bzip2_jll"; compat="1.0.9"),
     Dependency("LibCURL_jll"; compat="7.73,8"),
-    Dependency("Zlib_jll"),
+    Dependency("Zlib_jll"; compat="1.2.12"),
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
+# When using lld for AArch64 macOS, linking fails with
+#     ld64.lld: error: -dylib_current_version 10.4.3.1: malformed version
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
-               # When using lld for AArch64 macOS, linking fails with
-               #     ld64.lld: error: -dylib_current_version 10.4.3.1: malformed version
```

Comment:
Is the comment not relevant anymore?  It feels like it provided useful context as to why lld was disabled.

Reply:
The comment is still there, just one line earlier.